src/views/Message/CustomForm.tsx

```diff
@@ -11,21 +11,12 @@ import { application as application_interface } from "../../config/interface";
 
 const CustomForm = () => {
   const [typeformID, setTypeformID] = useState("NxomQTLD");
-  const { isLoading, isAuthenticated, getAccessTokenSilently } = useAuth0();
-  const [token, setToken] = useRecoilState(jwt);
+  const { isLoading, isAuthenticated } = useAuth0();
+  const [token] = useRecoilState(jwt);
```

Comment:
```suggestion
  const [token] = useRecoilValue(jwt);
```

When only needing to read a value and not have access to the setter, use `useRecoilValue`.